devops/scripts/check_dependencies.sh

```diff
@@ -12,12 +12,13 @@ if [ $? -ne 0 ]; then
   echo -e "\e[31m»»» ⚠️ Azure CLI is not installed! 😥 Please go to http://aka.ms/cli to set it up"
   exit
 fi
-
-echo -e "\n\e[96mChecking for Azure CLI extension(s)\e[0m..."
-az extension show -n azure-firewall > /dev/null 2>&1
-if [ $? -ne 0 ]; then
-  echo -e "\e[31m»»» ⚠️ Azure CLI azure-firewall extension is not installed! 😥 Please go to http://aka.ms/cli to set it up"
-  exit
+if [[ "$1" == *"azfirewall"* ]]; then
+  echo -e "\n\e[96mChecking for Azure CLI extension(s)\e[0m..."
+  az extension show -n azure-firewall > /dev/null 2>&1
+  if [ $? -ne 0 ]; then
+    echo -e "\e[31m»»» ⚠️ Azure CLI azure-firewall extension is not installed! 😥 Please go to http://aka.ms/cli to set it up"
```

Comment:
These messages could be more "devcontainer" friendly eg. "Please go to http://aka.ms/cli to set up or rebuild your devcontainer"

Reply:
You wrote this bit? :D